### ISSUE TITLE
fish: running `abed` removes the Git abbreviations

### ISSUE THREAD

**Issue by scy** [scy/dotfiles#6]
After calling `abed` in a fish instance, abbreviations like `gs` no longer work, at least not until you reload them (for example by running `gited`).

I guess this is because of the `set -g fish_user_abbreviations` at the start of `.config/fish/conf.d/_abbreviations.fish`?

**Comment by scy** [comment]
Argh, mentioned the wrong issue in the commit message. Reopening and force-pushing the revised commit. 🤦‍♂

**Comment by scy** [comment]
`WONTFIX`, I’m no longer using fish.
